documentation/docs/02-template-syntax/05-styles-and-classes.md

```diff
@@ -66,6 +66,36 @@ The `-global-` part will be removed when compiled, and the keyframe will then be
 </style>
 ```
 
+## :global
+
+To apply all styles after a certain point to a selector globally, use the `:global` modifier.
+
+```svelte
+<style>
+	:global {
+		div {
+			/* this will apply to every <div> in your application */
+			margin: 0;
+		}
+		p {
+			/* this will apply to every <p> in your application */
+			color: blue;
+		}
```

Comment:
```suggestion
		/* this will apply to every <div> in your application */
		div {...}
		
		/* this will apply to every <p> in your application */
		p {...}
```